CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.5.6
+
+* Update tax and benefit system initialization method in scripts.
```

Comment:
It's pretty hard for me as a client of OpenFisca parsers to understand how exactly that impacts me. The description makes it sound like I have to change my init calls, but it's a patch bump.
Please try to write the changelog from the point of view of the user, not from the producer :)